apps/cb-ceiling/server/dataFunctions/data_dailymodelcycle.js

```diff
@@ -139,7 +139,7 @@ dataDailyModelCycle = function (plotParams, plotFunction) {
     }
 
     var d;
-    if (diffFrom == null) {
+    if (diffFrom === null) {
```

Comment:
Actually, if (diffFrom == null) is equivilent to 
if (diffFrom === null || diffFrom === undefined) but it might not matter here.